### ISSUE TITLE
Free disk space not updated

### ISSUE THREAD

**Issue by davidak** [elementary/files#2035]
### What Happened?

When i hover over the root filesystem, it reports 6,3 GB free disk space, but more is already used and only 200 MB are actually free.

### Steps to Reproduce

1. Check free disk space in files

![Screenshot from 2022-04-28 19-34-41](https://user-images.githubusercontent.com/91113/165813566-3ec85c1e-33fc-4be8-b806-4f9b849f19c9.png)

2. Use some of that space
3. check disk space again. it has not changed
4. check with a different tool like `df -h` in terminal

```
[davidak@gaming:~]$ df -h
Filesystem      Size  Used Avail Use% Mounted on
devtmpfs        1.6G     0  1.6G   0% /dev
tmpfs            16G  477M   16G   3% /dev/shm
tmpfs           7.9G  7.7M  7.9G   1% /run
tmpfs            16G  480K   16G   1% /run/wrappers
/dev/sda2       450G  427G  2.1G 100% /
tmpfs            29G  153M   29G   1% /tmp
tmpfs           3.2G   84K  3.2G   1% /run/user/1000
```

the available space is different!

### Expected Behavior

the free disk space should be displayed in real time when i hover it

### OS Version

Other Linux

### Software Version

Latest release (I have run all updates)

### Hardware Info

Internal SATA SSD

**Comment by jeremypw** [comment]
Confirmed when the file operations are external.  When the file operation is performed with Files however, the device bookmark in the sidebar is updated.  Either way, showing "Properties" of the device does give the correct free space.

**Comment by jeremypw** [comment]
I think we will have to poll for external free space changes as I do not think there is a signal we can connect to.

**Comment by vjr** [comment]
Instead of polling, is it not possible to update the free space info when the tooltip is displayed? is something like a signal emitted when the tooltip is displayed?

**Comment by jeremypw** [comment]
@vjr Yes, that may be a better solution.  The only slight drawback would be that the diskspace indicator would not be updated until the tooltip was shown.
